.github/workflows/image-build.yaml

```diff
@@ -61,6 +64,8 @@ jobs:
               type=raw,value=latest,enable={{is_default_branch}}
 
           - registry: quay.io
+            build_language: go
+            bpf_build_wrapper: go
```

Comment:
This only needs the `build_language` I think

Reply:
You are right. That was how I had the original patch, but the `bpf_build_wrapper` line sneaked in when I was making sure I didn't introduce the clippy errors. I'll remove it.

Reply:
Sounds good thanks!